app/controllers/notification_controller.rb

```diff
@@ -14,20 +19,120 @@ def index
   #Need to modify create_monitor, right now does the same thing as index
   def create_monitor
 
-    #store user inputs: course number and email. Maybe also add a layer of validation of numbers and emails?
-    #"@" means globally accessible varaibles; available to all methods and corresponding rendered pages within the class
     #### Could also verify if emails and courseNumber are valid?
     @email = params.fetch("email")
-    @courseNum = params.fetch("courseNumber")
-    #courseNum, go to the public course search 
+    @courseNum = params.fetch("courseNum")
+    @key = "key-6e8f628f8c07bb415ef31325c2207dbf" 
```

Comment:
This value should be hidden in [an environment variable](https://chapters.firstdraft.com/chapters/792) and not be visible directly in the code.